web/themes/new_weather_theme/templates/block/block--weathergov-current-conditions.html.twig

```diff
@@ -3,128 +3,95 @@
+                                </p>
+                            </div>
+                        </div>
+                    </div>
 
-      <div class="display-flex flex-align-center margin-bottom-2">
-        {% if content.icon.icon %}
-        <div class="wx-icon margin-right-105 width-5 height-5">
-          <svg role="img" aria-hidden="true" class="width-full height-full">
-              <use xlink:href="{{ "/" ~ directory ~ "/assets/images/spritesheet.svg#" ~ content.icon.base }}"></use>
-            </svg>
-        </div>
-        {% endif %}
-        <p class="font-body-md margin-top-2px margin-bottom-0">{{ content.conditions.short }}</p>
-      </div>
-      <div class="tablet:margin-top-2 display-flex flex-row flex-align-center margin-bottom-05 border-top-1px border-base-light padding-top-1" aria-hidden="true">
-        <div class="margin-right-4">
-          <p class="margin-y-0">
-            <span class="font-mono-xs font-family-mono text-uppercase text-base">Humidity </span>
-            {{ "@humidity%" | t({ "@humidity": content.humidity }) }}
-          </p>
-        </div>
+                    <div class="display-flex flex-align-center margin-bottom-2">
+                        {% if content.icon.icon %}
+                            <div class="wx-icon margin-right-105 width-5 height-5">
+                                <svg role="img" aria-hidden="true" class="width-full height-full">
+                                    <use xlink:href="{{ "/" ~ directory ~ "/assets/images/spritesheet.svg#" ~ content.icon.base }}"></use>
+                                </svg>
+                            </div>
+                        {% endif %}
+                        <p class="font-body-md margin-top-2px margin-bottom-0">{{ content.conditions.short }}</p>
+                    </div>
+                </div>
 
-        <div class="wx-wind-speed margin-right-05 display-flex flex-align-center">
-          <p class="margin-y-0">
-            {#
-            If we want the space, we need to be explicit. Otherwise browsers
-            will sometimes strip it out. Like if, for example, a span is
-            followed by a non-text element such as a div.
-            #}
-            <span class="font-mono-xs font-family-mono text-uppercase text-base">Wind&nbsp;</span>
-            {% include '@new_weather_theme/partials/wind.html.twig'
-              with {
-                wind: content.wind
-              }
-            %}
-          </p>
+                <div class="grid-row tablet:margin-top-2">
+                    <div class="grid-col-12 tablet:grid-col-6 margin-bottom-05 tablet:padding-right-4">
+                        {%  include '@new_weather_theme/partials/observation-table.html.twig'%}
+                    </div>
+                    <div class="grid-col-12 tablet:grid-col-6">
+                        <div class="margin-top-3 tablet:margin-top-0 border-top-1px border-base-light padding-top-1">
```

Comment:
There is a some extra space right now, I am guessing because margins aren't collapsing, so we could get rid of the `margin-top-3 tablet:margin-top-0` here to make the spacing a little closer



![Screenshot comparing current PR and tighter spacing between weather variable table and the observation station information](https://github.com/weather-gov/weather.gov/assets/150970973/68c1c2eb-46b6-4f01-aa99-2c4cadc6efa8)

Reply:
https://github.com/weather-gov/weather.gov/pull/1285/commits/12281022d4a3b21167568782ba47e359ee8f1fa1